README.md

```diff
@@ -359,6 +362,7 @@ The `metadata` object in [`session/update`](#server--client-sessionupdate) has t
   - `album_artist?`: string | null
   - `album?`: string | null
   - `artwork_url?`: string | null
+  - `lyric_url?`: string | null
```

Comment:
should the client receive the lyrics from an HTTP location or should we instead send the lyrics as binary data ?